src/Autofac/Core/Activators/Reflection/ConstructorBinder.cs

```diff
@@ -154,7 +154,9 @@ public BoundConstructor Bind(IEnumerable<Parameter> availableParameters, ICompon
         private static MethodCallExpression ConvertPrimitiveType(Expression valueExpression, Type conversionType)
         {
             var changeTypeMethod = typeof(Convert).GetRuntimeMethod(nameof(Convert.ChangeType), new[] { typeof(object), typeof(Type) });
-            return Expression.Call(changeTypeMethod, valueExpression, Expression.Constant(conversionType));
+
+            // changeTypeMethod will always be non-null; Convert.ChangeType definitely exists.
```

Comment:
Interestingly enough, at least in early linker trimming [`Convert.ChangeType` was _not_ definitely always there.](https://github.com/autofac/Autofac/issues/842)

Reply:
Again, just to draw attention to this - `Convert.ChangeType` wasn't there in the past; and the `List<T>(int)` constructor may suffer the same fate. I don't think there's any action to take, but just something of which to be aware.